backend/core/src/seed.ts

```diff
@@ -0,0 +1,59 @@
+import dotenv from "dotenv"
+import "reflect-metadata"
+import { createConnection } from "typeorm"
+import { Listing } from "./entity/Listing"
+import { Listing as OldListing } from "@bloom-housing/core"
+import listingsLoader from "./lib/listings_loader"
+import { Unit } from "./entity/Unit"
+import { Attachment } from "./entity/Attachment"
+import { Preference } from "./entity/Preference"
+
+const skipped = ["id", "units", "attachments", "preferences"]
+const types = { units: Unit, attachments: Attachment, preferences: Preference }
+let l
```

Comment:
/pedantic, but is there any reason to declare `l` in this outer scope? it seems like it could be a `const` scoped to the `listing` loop instead?

Reply:
👍